<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcForm</class>
 <widget class="QWidget" name="CalcForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QDoubleSpinBox" name="dbspbox_price">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_raise_percent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价差:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="updown_percent">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_raise_percent_bfh">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_calcwin_start_price">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始价格:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_quantity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>161</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="lab_quantity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>买卖数量:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_fee_intro">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注:佣金 3/10000,最低5元
    印花税 1/1000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_result">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>fee ret</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_calc">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radio_up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>上涨</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>下跌</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_result_tag">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结果:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
